converters/google/convert.go

```diff
@@ -240,7 +244,7 @@ func (c *Converter) AddResourceChanges(changes []*tfjson.ResourceChange) error {
 // both fetch and mergeDelete. Supporting just one doesn't
 // make sense, and supporting neither means that the deletion
 // can just happen without needing to be merged.
-func (c *Converter) addDelete(rc *tfjson.ResourceChange) error {
+func (c *Converter) addDeleteResourceChange(rc *tfjson.ResourceChange) error {
```

Comment:
If the other method does switch to `addCreateOrUpdateOrNoop`, this should go back to `addDelete`

Reply:
Renamed back to `addDelete`.